### PR TITLE
Portfolio Mobile version project

### PR DESCRIPTION
**Pull request by anasbassi**
In this project we worked on Portfolio setup and mobile version project by setting up linters correctly, adding HTML and CSS files with a professional readme

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -31,6 +138,50 @@ ul {
   margin-bottom: 0;
 }
 
+.contact {
+  background-image: url(../images/Contact.png);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
```
Comment (Div685) [inline css/styles.css:147]: - [ ] [Required] Please Add proper margin to the footer (Contact) Section as shown in the Figma design. This makes your portfolio looks more beautiful.

File: index.html [hunk 2]
```diff
@@ -37,6 +37,119 @@ <h2 class="font connect">LET'S CONNECT </h2>
...
+        <li><button class="lang">css</button></li>
+        <li><button class="lang">javaScript</button></li>
+      </ul>
+      <button class="btn">See Project</button>
+    </div>
+  </section>
+  <section class="about font align">
+    <h2 class="intro font">About Myself</h2>
+    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="font connect">LET'S CONNECT </p>
```
Comment (Div685) [inline index.html]: - [ ] [Required] Please, Let's make sure to use the header tag `hx` on every sub-title such as this one.

File: index.html [hunk 3]
```diff
@@ -37,6 +37,119 @@ <h2 class="font connect">LET'S CONNECT </h2>
...
+      <li><a href=""><img src="images/twiter.svg" alt=""></a></li>
+      <li><a href=""><img src="images/N.svg" alt=""></a></li>
+      <li><a href=""><img src="images/M.svg" alt=""></a></li>
+      <li><a href=""><img src="images/github.svg" alt=""></a></li>
+      <li><a href=""><img src="images/two.svg" alt=""></a></li>
+    </ul>
+    <button class="btn">Get my resume</button>
+    <ul class="skills">
+      <li>
+        <h2>Languages</h2>
```
Comment (Div685) [inline index.html:128]: - [ ] [Required] Please, Add Drop-down Icon after the `Languages` heading as shown in the Figma design 
Lang_Screenshot from 2022-05-28 20-20-31

File: index.html [hunk 4]
```diff
@@ -37,6 +37,119 @@ <h2 class="font connect">LET'S CONNECT </h2>
...
+        <ul class="tray">
+          <li>
+            <img class="tray-img" src="images/javascript-logo.png" alt="">JavaSript</li>
+          <li><img class="tray-img" src="images/html-logo.png" alt="">HTML</li>
+          <li><img class="tray-img" src="images/css-logo.png" alt="">CSS</li>
+        </ul>
+      </li>
+      <li><h2>Frameworks</h2></li>
+      <li><h2>Skills</h2></li>
```
Comment (Div685) [inline index.html]: - [ ] [Required] Please, Add a Drop-down icon after the Frameworks and Skills heading.

File: index.html [hunk 5]
```diff
@@ -37,6 +37,119 @@ <h2 class="font connect">LET'S CONNECT </h2>
...
+    </ul>
+    <button class="btn">Get my resume</button>
+    <ul class="skills">
+      <li>
+        <h2>Languages</h2>
+        <ul class="tray">
+          <li>
+            <img class="tray-img" src="images/javascript-logo.png" alt="">JavaSript</li>
+          <li><img class="tray-img" src="images/html-logo.png" alt="">HTML</li>
+          <li><img class="tray-img" src="images/css-logo.png" alt="">CSS</li>
```
Comment (Div685) [inline index.html:134]: - [ ] [Required] Please Make the `JavaScript, HTML and CSS text` align with logo. And it's always better to check the typing errors (JavaSript -> JavaScript).
JS_Screenshot from 2022-05-28 20-25-44

**Review by Div685 — CHANGES_REQUESTED** [review]
Hi, **<@anasbassi>**, I'm Divyesh your reviewer for this PR :man_technologist: 

Good job making the Portfolio Mobile Version project,
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- ✔️ Linters are working :clap: 
- ✔️ Descriptive Readme file :clap: 
- ✔️ Descriptive PR Title. :clap:
- ✔️ Use of Semantic HTML.


## Required Changes ♻️
- [ ] [REQUIRED] It's always a best practice to check the *[Responsiveness](https://www.w3schools.com/html/html_responsive.asp)* of your website when working with a Mobile view. In your case you only have a Mobile view on your website Please make it responsive, It doesn't work well with mobile screen views. You can use Media queries to solve this. It's also advisable to use a Mobile-first CSS strategy where you give priority to mobile over desktop view.
**_Check the comments under the review._**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @anasbassi 👋🏻, 

Good job making the requested changes 💪🏻. Your portfolio looks awesome 😍.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**